tests/web/classes/test_blocks.py

```diff
@@ -688,3 +689,18 @@ def test_with_real_payload(self):
             }
         }
         self.assertDictEqual(input, CallBlock(**input).to_dict())
+
+
+# ----------------------------------------------
+# Header
+# ----------------------------------------------
+
+
+class HeaderBlockTests(unittest.TestCase):
+    def test_document(self):
+        input = {
+            "type": "header",
+            "block_id": "budget-header",
+            "text": {"type": "plain_text", "text": "Budget Performance"},
+        }
+        self.assertDictEqual(input, HeaderBlock(**input).to_dict())
```

Comment:
I know other existing tests don't do this but let's have the following as well. As mentioned above, Codecov detected this code path is not tested 
```python
self.assertDictEqual(input, Block.parse(input).to_dict())
```

Reply:
Thanks for the suggestion on how to test `Block.parse`. I've applied the update in commit 0ed0b1127bde635f3aec5da47f11f9d7b4bd9c4d. In the commit message details, I referenced the #775 pull request which adds the same coverage for the other blocks.